gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest_base.py.j2

```diff
@@ -56,7 +55,7 @@ class _Base{{ service.name }}RestTransport({{service.name}}Transport):
     {# TODO: handle mtls stuff if that is relevant for rest transport #}
     def __init__(self, *,
             host: str{% if service.host %} = '{{ service.host }}'{% endif %},
-            credentials: Optional[ga_credentials.Credentials] = None,
+            credentials: Optional[Any] = None,
```

Comment:
Please a comment in the code to clarify the reason that we have `Any` instead of `ga_credentials.Credentials`

Reply:
+1. If we could have a stronger type here, that would be great. Same for similar places below.

Reply:
addressed.